.browserslistrc

```diff
@@ -31,7 +31,7 @@ firefox 78
 ios_saf 12.2
 kaios 2.5
 op_mini all
-op_mob 73
+op_mob 80
```

Comment:
version 73 is not listed anymore
https://caniuse.com/ciu/comparison
https://browsersl.ist/#q=last+25+op_mob+versions